postgres/datadog_checks/postgres/util.py

```diff
@@ -1127,3 +1127,32 @@ def trim_leading_set_stmts(sql):
         {'name': 'locks.idle_in_transaction_age', 'type': 'gauge'},
     ],
 }
+
+# Column statistics from pg_statistic
+# Requires SELECT permissions on specific pg_statistic columns:
+#   `GRANT SELECT(starelid, staattnum, stawidth, stadistinct) ON pg_statistic TO datadog;`
+COLUMN_METRICS = {
+    'descriptors': [('schemaname', 'schema'), ('tablename', 'table'), ('attname', 'column')],
+    'metrics': {
+        'avg_width': ('column.avg_width', AgentCheck.gauge),
+        'n_distinct': ('column.n_distinct', AgentCheck.gauge),
+    },
```

Comment:
- We should verify whether the current metrics are optimal for use in recommendations. For example, column statistics can be used to calculate index selectivity before proposing an index. 
- We should check whether emitting tens of thousands of tags per database is problematic, as this would significantly exceed the scale we have handled previously.

I don’t have the answers, but it’s something we should check.

Reply:
> Should we consider that same scoping here?
We should get all table (i.e. relation) columns.